content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -756,6 +757,141 @@ WHERE  to_commit='pu60cdppae7rumf1lm06j5ngkijp7i8f';
 +--------------------+
 ```
 
+## `dolt_column_diff`
+
+The `dolt_column_diff` system table shows which columns and tables in the current database were changed in each commit 
+reachable from the active branch's HEAD. When multiple columns are changed in a single commit, there is one row in the 
+`dolt_column_diff` system table for each column, all with the same commit hash. Any staged or unstaged changes in the 
+working set are included with the value `WORKING` for their `commit_hash`.
```

Comment:
I thought we differentiated between `WORKING` and `STAGED`? Is that not true? Maybe I'm thinking of a different command.